.github/workflows/test.yml

```diff
@@ -5,9 +5,18 @@ jobs:
     name: Test
     strategy:
       matrix:
-        platform: [macos-latest, windows-2019]
+        platform: [macos-latest, windows-latest]
```

Comment:
If windows-latest works with the pagefile increase, we probably should just stick to it instead of forcing 2019 IMO